randovania/games/prime1/json_data/Phendrana Drifts.txt

```diff
@@ -1062,6 +1062,9 @@ Extra - aabb: [113.323105, -710.18396, 43.18039, 295.7474, -534.322, 117.33082]
                       # https://youtu.be/0V8nQpAi4fM
                       Slope Jump (Expert) or Use Grapple Beam
               Movement (Intermediate) and Use Grapple Beam
+              All of the following:
```

Comment:
Can you do this without triggering Thardus? If not, this needs to be "After Thardus"

Reply:
![image](https://github.com/randovania/randovania/assets/125271738/2b67c66d-4a8f-4343-9772-f505249f13b0)
Confirmed possible pre-thardus in game as well.